src/csharp/Microsoft.Spark/SparkContext.cs

```diff
@@ -386,5 +388,12 @@ private JvmObjectReference WrapAsJavaRDD(JvmObjectReference rdd)
                 "toJavaRDD",
                 rdd);
         }
+
+
+        /// <summary>
+        /// Returns t.
+        /// </summary>
+        /// <returns>The version of Spark on which this application is running</returns>
+        public static Version Version() => SparkEnvironment.SparkVersion;
```

Comment:
You should call the `version` defined in `SparkContext` on the Scala side.

Reply:
This has been revised